lib/configuration_methods.rb

```diff
@@ -141,8 +141,8 @@ def get_git_info
 
       git_cmd = `git log -1 --pretty="format:%H %ci"`
       if git_cmd =~ /^(\w+?)\s(.+)$/
-        @git_revision = $1
-        @git_update = $2.strip
+        @git_revision = Regexp.last_match[1]
+        @git_update = Regexp.last_match[2].strip
```

Comment:
I don't think that's an improvement, should be refactored to String#match instead.